app/src/main/java/com/droidhats/campuscompass/repositories/IndoorLocationRepository.kt

```diff
@@ -40,36 +43,40 @@ class IndoorLocationRepository private constructor(private val indoorLocationDao
             indoorLocationDao.deleteAllIndoor()
         }
         if (config == "debug" || indoorLocationDao.getOne().value == null) {
-            map.forEachBuilding { building ->
```

Comment:
we might need to check if this method is still being used

Reply:
true, It was not being used, so I removed it